src/scss/components/form/input-group/_addon.scss

```diff
@@ -1,4 +1,20 @@
 .Input-addOn {
-  flex: 0 0 auto;
```

Comment:
Not working at all on Chrome.
I think we need `flex: 0 0 auto;` and can delete `display: inline-block;`